backend/Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /code
 
 COPY poetry.lock pyproject.toml  /code/
 
-RUN pip install --no-cache-dir --upgrade "poetry==1.1.14"
+RUN pip install --no-cache-dir --upgrade pip poetry
```

Comment:
After this change there is an issue installing dependencies when running `make`. I get  `No module named 'filelock'`.

It seem like it's an issue with Poetry version 1.2.0.  

Version 1.15.0 works.

Reply:
Yeah, I changed it because I had several buil Issues too... Not sure why 🤔 Poetry bugs me 😅 I can revert!